### PR TITLE
Inform about skipping non-detected buildpack

### PR DESCRIPTION
**Pull request by dmathieu**

### CODE REVIEW

**Comment by tt** [comment]
Would it be better if this was an error? Is there a case for adding a buildpack that may not run? Worst case it could yield an empty slug.

Summoning @craigkerstiens and @jkutner for their feedback.

**Comment by craigkerstiens** [comment]
+1 for erroring, otherwise it may be unclear to the user. Even in the case when just vendoring in binaries some ack that detect is working feels correct.

File: bin/compile [hunk 1]
```diff
@@ -59,6 +59,9 @@ for BUILDPACK in $(cat $1/.buildpacks); do
       if [ -x $dir/bin/release ]; then
         $dir/bin/release $1 > $1/last_pack_release.out
       fi
+    else
+      echo "Couldn't detect any framework for this buildpack. Skipping."
```
Comment (tt) [inline bin/compile:63]: "Skipping" might not make sense when it's exiting.

Comment (dmathieu) [inline bin/compile:63]: Fixed in #4

**Pull request merged**